src/main/java/org/spongepowered/api/plugin/Plugin.java

```diff
@@ -82,5 +82,13 @@
      */
     String dependencies() default "";
 
+    /**
+     * Gets the prefixes that can be prepended to a command to indicate this
+     * plugin's version of said command. This must have at least one item.
```

Comment:
There doesn't have to be at least one item if you state that the default prefix is to use the plugin ID, which is most likely what plugin devs would use as the primary prefix anyway.